app/models/participant_profile/ecf.rb

```diff
@@ -146,7 +146,10 @@ def update_analytics
   def sync_status_with_induction_record
     induction_record = induction_records.latest
     induction_record&.update!(induction_status: status) if saved_change_to_status?
-    induction_record&.update!(mentor_profile:) if saved_change_to_mentor_profile_id?
+    if saved_change_to_mentor_profile_id?
```

Comment:
Would it be better to encapsulate these in the `InductionRecord`/`ParticipantProfile` models and touch the user through lifecycle callbacks to cover other use cases? doing it this way means if the `induction_record.mentor_profile` or `participant_profile.mentor_profile` is updated elsewhere it won't touch the user?